nuxeo-arender-rendition-apb/files/arender_document_converter_logback.xml

```diff
@@ -1,13 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration>
-    <property name="logging.file" value="/logs/arender-document-converter/${POD_NAME}.log" />
+    <property name="logging.directory" value="/logs/arender-document-converter" />
+    <property name="logging.file" value="${logging.directory}/${POD_NAME}.log" />
     <appender name="SERVER" class="ch.qos.logback.core.rolling.RollingFileAppender">
         <file>${logging.file}</file>
         <encoder>
             <pattern>%date %level [%thread] %logger{10} [%file:%line] %msg%n</pattern>
         </encoder>
         <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>arender-taskconversion.%i.log.zip</fileNamePattern>
+            <fileNamePattern>${logging.directory}/arender-document-converter.%i.log.zip</fileNamePattern>
```

Comment:
To differentiate different zip from different pod, maybe you can also inform the pod name there in order to keep the same name to the log file
Suggestion if it is possible :
<fileNamePattern>${logging.directory}/${POD_NAME}.%i.log.zip</fileNamePattern>

Same the others config files